src/tasks/tasks.service.ts

```diff
@@ -29,4 +33,22 @@ export class TasksService {
       );
     }
   }
+
+  async findTaskById(id: number): Promise<Task | null> {
+    try {
+      const task = await this.tasksRepository.findOneTask(id);
```

Comment:
```suggestion
      const task = await this.tasksRepository.findOneOrFail(id);
```

Can use this builtin repository method. Not Found error exception need not be handled explicitly.